apps/reservation.rb

```diff
@@ -7,6 +7,7 @@
 
       PostToSlack.say(:reservation_bot, @reservation.success_message) if production?
       status(200)
+      {'responseText'=> 'Rservation created'}.to_json
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.